.yarnrc.yml

```diff
@@ -62,25 +62,34 @@ packageExtensions:
   "@signal-noise/stylelint-scales@2.0.3":
     peerDependencies:
       postcss: "*"
-  "@wordpress/commands@0.6.2":
+  "@wordpress/block-editor@12.5.0":
```

Comment:
Unfortunately, it seems like these peer dependency issues are regressing. About two years ago, I got everything fixed internal to Gutenberg, but seems like that's stopped happening again, leaving these issues scattered about everywhere.